clipctl

```diff
@@ -1,25 +1,53 @@
 #!/usr/bin/env bash
 
+: "${CM_DIR:="${XDG_RUNTIME_DIR-"${TMPDIR-/tmp}"}"}"
+
 if [[ -z $1 ]] || [[ $1 == --help ]] || [[ $1 == -h ]]; then
     cat << 'EOF'
 clipctl provides controls for the clipmenud daemon.
 
 You can temporarily disable clip collection without stopping clipmenud entirely
 by running "clipctl disable". You can then reenable with "clipctl enable".
+
+Commands:
+    enable: enable clip collection
+    disable: disable clip collection
+    status: returns "enabled" or "disabled"
+    toggle: toggles clip collection
+    version: returns major version
+    cache-dir: returns the directory used for caching
 EOF
     exit 0
 fi
 
-_CLIPMENUD_PID=$(pgrep -u "$(id -u)" -nf 'clipmenud$')
+clipmenud_pid=$(pgrep -u "$(id -u)" -nf 'clipmenud$')
 
-if [[ -z "$_CLIPMENUD_PID" ]]; then
-    echo "clipmenud is not running"
-    exit 2
-fi
+case $1 in
+    enable|disable|toggle|status)
+        if [[ -z "$clipmenud_pid" ]]; then
+            echo "clipmenud is not running"
+            exit 2
+        fi
+        ;;
+esac
+
+major_version=6
+cache_dir=$CM_DIR/clipmenu.$major_version.$USER
+status_file=$cache_dir/status
 
 case $1 in
-    enable) kill -USR2 "$_CLIPMENUD_PID" ;;
-    disable) kill -USR1 "$_CLIPMENUD_PID" ;;
+    enable) kill -USR2 "$clipmenud_pid" ;;
+    disable) kill -USR1 "$clipmenud_pid" ;;
+    status) cat $status_file ;;
```

Comment:
This one should also be quoted, since otherwise things will go wonky if `$CM_DIR` has spaces or similar.

Reply:
Done